linux_os/guide/system/software/integrity/crypto/ssh_client_rekey_limit/rule.yml

```
@@ -0,0 +1,43 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure session renegotiation for SSH client'
+
+description: |-
+    The <tt>RekeyLimit</tt> parameter specifies how often
+    the session key is renegotiated, both in terms of
+    amount of data that may be transmitted and the time
+    elapsed. To decrease the default limits, put line
+    <tt>RekeyLimit {{{ sub_var_value("var_ssh_client_rekey_limit_size") }}} {{{ sub_var_value("var_ssh_client_rekey_limit_time") }}}</tt> to file <tt>/etc/ssh/ssh_config.d/02-rekey-limit.conf</tt>.
+    Make sure that there is no other <tt>RekeyLimit</tt> configuration preceding
+    the <tt>include</tt> directive in the main config file
+    <tt>/etc/ssh/ssh_config</tt>. Check also other files in
+    <tt>/etc/ssh/ssh_config.d</tt> directory. Files are processed according to
```

I would add something like:

```suggestion
    <tt>/etc/ssh/ssh_config.d</tt> directory. Files are processed according to lexicographical order of their names
```